kl_conferences/serializers/server_node_serializer.py

```diff
@@ -11,14 +14,24 @@ class Meta:
         fields = '__all__'
 
 
+class HostnameField(serializers.CharField):
+    max_length = 128
+
+    def to_internal_value(self, data):
+        data = str(data)
+        if not re.match(r'^([\w\d]+\.)+\w+(:\d+)?$', data):
```

Comment:
no "-" allowed in hostname?

Reply:
We can leave it for now, hostname will be composed from sponsorname+number.bbb.teleklasa.lokalni.pl (eg. oktawave1.bbb.teleklasa.lokalni.pl)

Reply:
feel free to commit a fix